corehq/form_processor/serializers.py

```diff
@@ -154,7 +154,7 @@ class CommCareCaseSQLAPISerializer(serializers.ModelSerializer):
     date_modified = serializers.DateTimeField(source='modified_on')
     properties = serializers.JSONField(source='get_properties_in_api_format')
     server_date_modified = serializers.DateTimeField(source='server_modified_on')
-    server_date_opened = serializers.DateTimeField(source='opened_on')
+    server_date_opened = serializers.DateTimeField(source='server_opened_on')
```

Comment:
Shall we rename this serialized property then to accurately reflect the value ("server_date_opened" -> "opened_on"), or are you suggesting that we remove it altogether?